<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModulSetting</class>
 <widget class="QWidget" name="ModulSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>501</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Setting modul :</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>IP modul</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="le_ip_modul">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>192.168.1.123</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>kanal-1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="chk_kanal1">
           <property name="text">
            <string>aktif</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>kanal-2</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>kanal-3</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>kanal-4</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QCheckBox" name="chk_kanal2">
           <property name="text">
            <string>aktif</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QCheckBox" name="chk_kanal3">
           <property name="text">
            <string>aktif</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QCheckBox" name="chk_kanal4">
           <property name="text">
            <string>aktif</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="4">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="11" column="2">
          <widget class="QCheckBox" name="chk_fasa_aktif">
           <property name="text">
            <string>aktif</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>trigger fasa</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>[kanal]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>[set aktif]</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>[Acc - mV/G]</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="le_acc1"/>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="le_acc2"/>
         </item>
         <item row="6" column="3">
          <widget class="QLineEdit" name="le_acc3"/>
         </item>
         <item row="7" column="3">
          <widget class="QLineEdit" name="le_acc4"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Parameter Vibrasi (modul) :</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>F-max (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>request lines</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cb_lines">
           <item>
            <property name="text">
             <string>100</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3200</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cb_fstop">
           <item>
            <property name="text">
             <string>200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>700</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>900</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2500</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5000</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10000</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Sampling rate</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Waktu sampling</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Jumlah data</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="labelSamplingRate">
           <property name="text">
            <string>Sample/s</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="labelDataCount">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="labelETA">
           <property name="text">
            <string>0s</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>F-min (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cb_fstart">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>60</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>120</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="pb_update_set_modul">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Update Setting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
